src/legacy/status_im/mobile_sync_settings/core.cljs

```diff
@@ -6,11 +6,15 @@
     [legacy.status-im.multiaccounts.update.core :as multiaccounts.update]
     [legacy.status-im.utils.mobile-sync :as utils]
     [legacy.status-im.wallet.core :as wallet]
-    [status-im.contexts.chat.home.add-new-contact.events :as add-new-contact]
     [status-im.navigation.events :as navigation]
     [taoensso.timbre :as log]
     [utils.re-frame :as rf]))
 
+(rf/defn set-new-identity-reconnected
```

Comment:
can we add this new effect in it's rightful home not in `legacy` folder?
I understand you might need to make more changes to have the cofx in the event below, but if we can at least reduce the code added to `legacy` it would be great. 👍